examples/root-example/main.tf

```
@@ -1,47 +1,32 @@
-variable "region" {
-  default = "us-central1"
+provider "google" {
+  version = "~> 3.0"
+
+  region  = var.region
+  project = var.project
 }
 
-variable "project" {}
+provider "google-beta" {
+  version = "~> 3.0"
 
-provider "google" {
-  region  = "${var.region}"
-  project = "${var.project}"
+  region  = var.region
+  project = var.project
 }
 
-provider "google-beta" {
-  region  = "${var.region}"
-  project = "${var.project}"
+provider "random" {
+  version = "~> 2.2"
 }
 
-module "tfe-cluster" {
-  source           = "hashicorp/terraform-enterprise/google"
-  version          = "0.1.0"
-  credentials_file = "auth-file-123456678.json"
-  region           = "${var.region}"
-  zone             = "${var.region}-a"
-  project          = "${var.project}"
-  domain           = "example.com"
-  dns_zone         = "example"
-  public_ip        = "1.2.3.4"
-  certificate      = "https://www.googleapis.com/compute/v1/project/terraform-test/global/sslCertificates/tfe"
-  ssl_policy       = "tfe-ssl-policy"
-  subnet           = "tfe-subnet"
-  frontend_dns     = "tfe-cluster"
-
-  primary_count   = "3"
-  secondary_count = "2"
-
-  license_file = "customer.rli"
+provider "template" {
+  version = "~> 2.1"
 }
 
-output "tfe-cluster" {
-  value = {
-    application_endpoint         = "${module.tfe-cluster.application_endpoint}"
-    application_health_check     = "${module.tfe-cluster.application_health_check}"
-    installer_dashboard_password = "${module.tfe-cluster.installer_dashboard_password}"
-    installer_dashboard__url     = "${module.tfe-cluster.installer_dashboard_url}"
-    primary_public_ip            = "${module.tfe-cluster.primary_public_ip}"
-    encryption_password          = "${module.tfe-beta.encryption_password}"
-  }
+module "tfe-cluster" {
+  source = "../.."
```

we probably still want this source going thru the registry instead of as a file system
```suggestion
  source = "hashicorp/terraform-enterprise/google"
```
with whatever version we're on `  version          = "0.1.0"`

---

Resolved by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/45/commits/1ba6a8653c40a7933eb7c21396d11d1e2c18ac1a.